application/cms/forms.py

```diff
@@ -0,0 +1,28 @@
+from flask_wtf import Form
+from wtforms import StringField
+from wtforms import validators
+
+
+class PageForm(Form):
```

Comment:
What I would do with these forms is just subclass FlaskForm, remove constructor and validator.

The disabling of csrf we can do from base config (and at the moment for heroku csrf not working for some reason with chrome and will have to be disabled globally anyway)

The validator isn't doing any work can be removed and then when we need more validation we'll create validators that can be added to validators list as we are with DataRequired.